invoice.html

```diff
@@ -0,0 +1,49 @@
+<div class="firmBox">
```

Comment:
jesli masz plik ktory nie zawiera pelnego html, a jest tylko templatem ktory zostanie wstawiony gdzie indziej, to powinien miec na koncu slowo template. w tym wypadku
invoice.template.html